README.md

```diff
@@ -67,7 +67,9 @@ This branch is used from time-to-time for experimental features under heavy deve
 
 ## Installation
 
-### Please follow ALL steps in this README if you encounter any issues with installation before seeking support.
+**Please Note**: Prior to asking for support for any installation issue follow ALL the steps outlined below:

```

Comment:
I believe this change doesn't really provide any more clarity / readability than the original sentence. 

It has extra verbosity via "prior to" which could be replaced with "before".
The double use of `:` is a punctuation error as well.